makefiles/tools/pic32prog.inc.mk

```diff
@@ -0,0 +1,42 @@
+# pic32prog flasher
+# =================
+#
+# https://github.com/sergev/pic32prog
+#
+# Allow flashing pic32 boards using:
+#  * Microchip PICkit2
+#  * Microchip PICkit3 with script firmware
+#
+#
+# PICkit-3
+# --------
+#
+# Changing the firmware to 'scripting mode'
+# Should be done from a Windows computer as described here
+#
+# http://ww1.microchip.com/downloads/en/DeviceDoc/PICkit3%20Programmer%20Application%20v3.10.zip
+#
+# ### Setup summary
+#
+# * Extract PICkit3 Programmer Application Setup v3.10.zip
+# * setup.exe
+# * Run PICkit3
+# * Tools/Download PICkit Operating System
+#   * Select C://Program Files/Microchip/PICkit3/PK3OSV020005.hex
+#
```

Comment:
Maybe add a reference to your tutorial somewhere in here instead?

Reply:
I did not put a reference here as the documentation was not merged. But I could wait for the documentation and update this.